src/Makevars.win

```diff
@@ -14,6 +14,11 @@ rpolars_ext_binary = "$(RPOLARS_RUST_SOURCE)/target/$(TARGET)/release/librpolars
 all: C_clean
 
 $(SHLIB): $(STATLIB)
+# CRAN policy forbids using $HOME so we build in the current working dir.
+# This makes things really slow because we have to reinstall all dependencies
+# for each build. Comment this out for non-CRAN releases.
+CARGOTMP=$(PWD)/.cargo
```

Comment:
`$(PWD)` may not work here.
`$(CURDIR)` should be use in Makefiles.
https://stackoverflow.com/questions/52437728/bash-what-is-the-difference-between-pwd-and-curdir

Reply:
Thank you for seeing this,  I will revisit this next week